flink-runtime/src/test/java/org/apache/flink/runtime/operators/chaining/ChainTaskTest.java

```diff
@@ -174,7 +185,68 @@ public void testFailingMapTask() {
 			Assert.fail(e.getMessage());
 		}
 	}
-	
+
+	@Test
+	public void testOutputInCloseMethod_batchTask() {
```

Comment:
The test methods should conform to standard java naming patterns, i.e. no underscore.

Reply:
fixed